addons/web/static/src/views/fields/datetime/datetime_field.js

```diff
@@ -343,6 +345,10 @@ export const dateTimeField = {
             ),
         },
     ],
+    extractProps: ({ attrs, options }, dynamicInfo) => ({
+        ...dateField.extractProps({ attrs, options }, dynamicInfo),
+        showTime: archParseBoolean(options.showTime ?? true),
```

Comment:
```suggestion
        showTime: archParseBoolean(options.show_time, true),
```
In archs, the convention is to snake case attributes/options